src/clr-angular/wizard/_variables.wizard.scss

```diff
@@ -20,6 +20,8 @@ $clr-wizard-box-shadow: 0 $clr-rem-1px (2 * $clr-rem-1px) (2 * $clr-rem-1px) hsl
 $clr-wizard-step-nav-border-color: hsl(0, 0%, 89%) !default;
 $clr-wizard-border-radius: $clr-global-borderradius !default;
 $clr-wizard-stepnav-active-border-radius: $clr-wizard-border-radius !default;
+$clr-wizard-stepnav-error-color: $clr-color-danger-800 !default;
```

Comment:
This value needs to be added/overridden in the dark theme.

Reply:
The icon color came from the `is-error` class, which has the light theme color. Should I wait for the https://github.com/vmware/clarity/pull/3838 pull request to be merged or I should override the icon color?

Reply:
You should be able to add that variable to the dark theme sass and override it there.

Reply:
The variable is now in the dark theme sass file and the icon respects the dark theme color also.